core/src/main/java/org/elasticsearch/discovery/zen/ping/unicast/UnicastZenPing.java

```diff
@@ -64,7 +64,7 @@
 
     public static final String ACTION_NAME = "internal:discovery/zen/unicast";
 
-    public static final int LIMIT_PORTS_COUNT = 1;
+    public static final int LIMIT_PORTS_COUNT = 10;
```

Comment:
This is really a nocommit. We should only do 10 i think when its the localhost default. Just trying to start simple :)